### PR TITLE
[Task] Add metadata to image version

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request
Part of https://github.com/pimcore/studio-ui-bundle/issues/352

### CODE REVIEW

File: src/Version/Schema/CustomMetadataVersion.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Version\Schema;
+
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+
```
Comment (lukmzig) [inline src/Version/Schema/CustomMetadataVersion.php:21]: Can it be internal? :)

**Review by lukmzig — APPROVED** [review]
One tiny thing, the rest LGTM :)

File: src/Version/Schema/CustomMetadataVersion.php (hunk 1 above)
Comment (mattamon) [reply src/Version/Schema/CustomMetadataVersion.php:21]: Good point since we do not give it to the outer world it can be internal yes :)

Comment (mattamon) [reply src/Version/Schema/CustomMetadataVersion.php:21]: Question is, should we remove it then from every schema that has the additional attributes, I guess so right?

Comment (lukmzig) [reply src/Version/Schema/CustomMetadataVersion.php:21]: Hmmm I guess that would make sense, yeah

**Pull request merged**